### PR TITLE
Add 2 test cases for tunnel QoS remapping

### PR DESCRIPTION
**Pull request by bingwang-ms**
### Description of PR

Summary:
This PR is to add two test cases for verifying tunnel QoS remapping.
Test plan: https://github.com/Azure/sonic-mgmt/pull/5508

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
This PR is to add two test cases for tunnel QoS remapping.
1. TC1 test_encap_dscp_rewrite
The test is to verify the dscp rewriting of encapped packets.
2. TC2 test_bounced_back_traffic_in_expected_queue
The test case is to verify the encapped packet is mapped to the correct queue
 
#### How did you do it?
Please see test plan.

#### How did you verify/test it?
Verified on a dualtor testbed.
```
collected 2 items                                                                                                                                                                                     

qos/test_tunnel_qos_remap.py::test_encap_dscp_rewrite PASSED                                                                                                                                    [ 50%]
qos/test_tunnel_qos_remap.py::test_bounced_back_traffic_in_expected_queue PASSED                                                                                                                [100%]
```
#### Any platform specific information?
No.
#### Supported testbed topology if it's a new test case?
Dualtor testbed specified.

### CODE REVIEW

File: tests/qos/test_tunnel_qos_remap.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    ipinip_packet = simple_ipv4ip_packet(
+                eth_dst=active_tor_mac,
+                eth_src=standby_tor_mac,
+                ip_src=standby_tor_ip,
+                ip_dst=active_tor_ip,
+                ip_dscp=outer_dscp,
+                ip_ecn=ecn,
+                inner_frame=pkt[IP]
+            )
+    pkt.ttl += 1
```
Comment (neethajohn) [inline tests/qos/test_tunnel_qos_remap.py:55]: Why is this needed?

Comment (bingwang-ms) [reply tests/qos/test_tunnel_qos_remap.py:55]: It's because the TTL of inner frame of encapsulated packet is decreased by 1. So we minus 1 at line 45. Then we need to recover it after building the ipinip packet.

**Review by neethajohn — APPROVED** [review]
